migration/incremental/028-remove-trace-create-temp-table-func.sql

```diff
@@ -0,0 +1,2 @@
+-- We added more generic function to be used by both metrics and traces so removing traces specific one
+DROP FUNCTION IF EXISTS _ps_trace.ensure_trace_ingest_temp_table(text, text) CASCADE;
```

Comment:
you need to remove this function from the idempotent scripts too or it will just get recreated

Reply:
Actually removing this function will break older version of Promscale. For now I'll just mark it as deprecated and later we can decide what to do here (I think we need more broad discussion about extension & connector version compatibility...)